pylint_odoo/checkers/no_modules.py

```diff
@@ -213,6 +213,13 @@
     ),
 })
 
+ODOO_MSGS.update({
```

Comment:
The special variable `ODOO_MSGS` is used for "Checks not oca just vx to avoid oca conflicts" (see comment above)
But for this check will be migrated to OCA project, then you could use the default global variable `ODOO_MSGS`